File: Models/AlertaSasmex.cs
using System;

namespace DetectorSismos.Models
{
    /// <summary>
    /// Representa una alerta del Sistema de Alerta Sísmica Mexicano (SASMEX/CIRES).
    /// Fuente: https://rss.sasmex.net
    /// </summary>
    public class AlertaSasmex
    {
        public string Id { get; set; } = string.Empty;
        public DateTime FechaHora { get; set; }
        public string Evento { get; set; } = string.Empty;
        public string Severidad { get; set; } = "Moderada";
        public string Descripcion { get; set; } = string.Empty;
    }
}

File: sasmex-android/app/src/main/java/mx/cires/sasmex/android/data/AlertaSasmex.kt
package mx.cires.sasmex.android.data

import java.util.Date

/**
 * Representa una alerta del Sistema de Alerta Sísmica Mexicano (SASMEX/CIRES).
 * Fuente: https://rss.sasmex.net
 */
data class AlertaSasmex(
    val id: String,
    val fechaHora: Date,
    val evento: String,
    val severidad: String,
    val descripcion: String
) {
    val esMayor: Boolean get() = severidad.contains("Mayor", ignoreCase = true)
    val esMenor: Boolean get() = severidad.contains("Menor", ignoreCase = true)
}
